nbval/plugin.py

```diff
@@ -234,7 +238,8 @@ def setup(self):
         else:
             kernel_name = self.nb.metadata.get(
                 'kernelspec', {}).get('name', 'python')
-        self.kernel = RunningKernel(kernel_name, str(self.fspath.dirname))
+        self.kernel = RunningKernel(
+            kernel_name, self.config.option.nbval_kernel_startup_timeout, str(self.fspath.dirname))
```

Comment:
If the above change is made, then also:
```suggestion
            kernel_name,
            cwd=str(self.fspath.dirname),
            startup_timeout=self.config.option.nbval_kernel_startup_timeout, 
        )
```